Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -12,7 +12,7 @@ param (
 Write-Verbose "Entering script PowerShellOnTargetMachines.ps1" -Verbose
 Write-Verbose "environmentName = $environmentName" -Verbose
 Write-Verbose "resourceFilteringMethod = $resourceFilteringMethod" -Verbose
-Write-Verbose "machineFilter = $machineFilter" -Verbose
+Write-Verbose "machineNames = $machineNames" -Verbose
```

Comment:
We might be retaining same name for compat. But to use machineNames as variable seem confusing especially when it can also contain tags. Do you want to assign it to $machineFilter variable within .ps1 and use $machineFilter everywhere? That gives better readability.